backend/src/middleware/validUrlMiddleware.js

```diff
@@ -0,0 +1,18 @@
+const validURL = str => {
+  const isValid = str.match(/^(?:https?:\/\/)(?:[^@\n])?(?:www\.)?([^:/\n?]+)/g)
+  return !!isValid
+}
+
+export default {
+  Mutation: {
+    CreateSocialMedia: async (resolve, root, args, context, info) => {
```

Comment:
@mattwr18 what about `UpdateSocialMedia`? :slightly_smiling_face: Or do we always want to Delete+Create?

Reply:
I was talking to @Tirokk about this this morning.
The ticket doesn't say anything about updating or deleting social media
So, I would probably create two separate issues for those use cases, which can be separately estimated and potentially would be beginner-friendly tickets for people to work on.

Reply:
@mattwr18 OK cool! That's a solution as well. Just make sure we don't forget these things. :+1:

Reply:
I can take responsibility for adding the tickets :smile_cat: